### DISCUSSION TITLE
image in markdown can't load

### DISCUSSION THREAD

**Discussion by hiktoop** [community/community#67565] [Mobile]
Images use relative address can't be loaded in a markdown file.

**Comment by stevepopovich** [comment]
Hi @hiktoop, do you have an example of this?

**Reply by hiktoop** [reply]
picture upload error, how can i give you my example?

**Reply by hiktoop** [reply]
i have sent it to your gmail.

**Comment by Manuel36791** [comment]
Check the link or address of the image sometimes it is copied and pasted in a wrong way

**Reply by hiktoop** [reply]
![IMG_20230920_011442.jpg](https://github.com/community/community/assets/72680955/4017bb16-b641-4475-ba2a-6b7876c08a28)

![IMG_20230920_011501.jpg](https://github.com/community/community/assets/72680955/6e57562f-0295-41f7-bf39-e666135e2028)

![IMG_20230920_011519.jpg](https://github.com/community/community/assets/72680955/55516a07-4c75-4707-8f7d-4b9c63b8fd9c)

**Reply by hiktoop** [reply]
something like this

**Reply by stevepopovich** [reply]
HI @hiktoop, Did you try naming your images? This can also be useful for a11y users as an alt description. So instead of
`[](link-to-photo.com)` to `[description-of-image](link-to-photo.com)`

Also, if you could link to these files, that would be useful. Thanks!

**Reply by hiktoop** [reply]
I've tried, it doesn't work

**Reply by hiktoop** [reply]
I think it is something about github file format in webpage
